pennylane_lightning/core/src/lightning_qubit.toml

```diff
@@ -0,0 +1,125 @@
+schema = 1
+
+[device]
+name = "lightning.qubit"
+precision = ['float32', 'float64']
+
+[operations]
+observables = [
+	"NamedObs",
+	"HermitianObs",
+	"TensorObs",
+	"HamiltonianObs",
```

Comment:
Do these names need to directly match what is the class-name in the CPP layer? If so, we may need to update these to what is here: https://github.com/PennyLaneAI/pennylane-lightning/blob/master/pennylane_lightning/core/src/simulators/lightning_qubit/observables/ObservablesLQubit.hpp

Also, we should also include the SparseHamiltonian object.

Reply:
There is a misconception here. This file relates more to Catalyst than PL-lightning. These are the observables supported by Catalyst. We do not support SparseHamiltonian.

Reply:
I feel like this misconception is appearing more often than not. I suggest if this will not represent a lot of work for you, you move these files to a directory called catalyst (or something equivalent). This way it will always be clear that these are catalyst "configuration" files.

Reply:
Yep, that works for me. Thanks @dime10 
With that we can eventually move to replace the included sets with the data provided by the TOML, and import them at the Python layer for a single source of truth.

Reply:
Files have been moved. https://github.com/PennyLaneAI/pennylane-lightning/pull/566/commits/4df3f8486bc76867eead81866965e92842e28735